recipes/champ/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "champ" %}
+{% set version = "2.0.2" %}
+{% set sha256 = "2e24d8fea58bf65bb13ec646cbb7404c4fc780ca4c07d5eb03c9aca9cb1609bb" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/ce/25/b7cba31f1781a7ddbcafb8ff8e96357ffa98d41704ccf7e19bcfe928649a/champ-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 0
+  skip: True  # [win]
+
+
+requirements:
+  build:
+    - pkg-config
```

Comment:
Does this contain any compiled code? I don't see it in the upstream repository?

Reply:
the champ package doesn't contain any compiled code.  one of the dependencies does (louvain ) so I had included that in there .  Can I remove it?  Thanks.

Reply:
Yes you can remove all of the `build` dependencies. Each package only cares about its direct dependencies.

Reply:
removed all build dependencies